recipes/sagemaker-pytorch-serving-container/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "sagemaker-pytorch-serving-container" %}
+{% set version = "1.1.1" %}
+{% set commit = "1ba29db3c90e3a71a708319693372ec2f51ef4fe" %}
```

Comment:
What is this commit line for?

Reply:
@scopatz forgot to remove it from the recipe -- it's good now.